Assets/MRTK/Core/Providers/BaseSpatialObserver.cs

```diff
@@ -40,6 +40,13 @@ public abstract class BaseSpatialObserver : BaseDataProvider<IMixedRealitySpatia
         [System.Obsolete("Call Service instead.")]
         protected IMixedRealitySpatialAwarenessSystem SpatialAwarenessSystem => Service;
 
+        protected GameObject observedObjectParent = null;
```

Comment:
can this stay `private`? the property can be changed if needed, but it feels better to generally keep fields `private` imo

Reply:
I would like it to be protected so that the implementation can opt to control / change it.

Reply:
on second thought, you make a good point. the only scenario i can imagine is pretty contrived